src/esp/physics/RigidBase.h

```diff
@@ -63,7 +63,13 @@ enum class MotionType {
    * Default behavior of @ref RigidObject with a physics simulator defined. See
    * @ref BulletRigidObject.
    */
-  DYNAMIC
+  DYNAMIC,
+
+  /**
+   * The object is not simulated or active in the collision world, but can be
+   * updated kinematically.
+   */
+  VISUALIZATION_ONLY
```

Comment:
At some point, we're going to add the capability for users to configure what collides with what. I'm talking about collision filtering. For example, we already configure different parts of the fetch to either collide with the static environment or not: https://github.com/ASzot/p-viz-plan/pull/2 .

Let's fast-forward to the day in the future where we have this feature. How is a VISUALIZATION_ONLY object different from a KINEMATIC object that has been configured not to collide with anything? Is this separate type even needed?